en_US/advanced/acl-mongodb.md

```diff
@@ -267,5 +267,8 @@ You can use the following placeholders in ACL queries, and EMQ X Broker will aut
 
 ::: tip
 MongoDB ACL rules need to use the above data structures strictly.
-All rules added in MongoDB ACL are **allow** rules, which can be used with `acl_nomatch=deny` in` etc/emqx.conf`.
-:::
\ No newline at end of file
+All rules added in MongoDB ACL are **allow** rules. It means a force whitelist.
+The ACL check would be failed when ACL rule list is not empty for specific topic.
+And the ACL chain will be terminated.
+It's recommended to set MongoDB ACL be the last one of ACL chain.
```

Comment:
```suggestion
All rules added in MongoDB ACL are all **allow** rules. i.e. a whitelist.

When the rules list of a given topic is empty, EMQ X continues to check the next auth/ACL plugin.
Otherwise the check returns immediately without proceeding to the next auth/ACL plugins.

When more than one auth/ACL plugins are in use, it is recommended to position MongoDB ACL after other auth/ACL plugins in the enabled plugins list.
```